src/notebookRunGroups/util/cellMetadataHelpers.ts

```diff
@@ -27,6 +27,6 @@ export function updateCellRunGroupMetadata(cell: vscode.NotebookCell, newGroupVa
     // Perform our actual replace and edit
     const wsEdit = new vscode.WorkspaceEdit();
     const notebookEdit = vscode.NotebookEdit.updateCellMetadata(cell.index, newMetadata);
-    wsEdit.set(cell.document.uri, [notebookEdit]);
```

Comment:
So this was the cause of #47 when Rich updated the PowerToys to new API changes. I think every team member has now gotten this same thing wrong once (I did it for sure). Still feels like an API slip up to me.